canvas/migrations/0026_auto_20230409_1922.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.14 on 2023-04-10 02:22
+
+from django.db import migrations
```

Comment:
you should only have 1 migration. Do these steps to make them into one.

1. revert these 2 migrations.
2. delete the migrations files
3. run makemigration and migrate again